### ISSUE TITLE
"npm run build" overwrites source code

### ISSUE THREAD

**Issue by rojlarge** [kuali/research-coi#16]

**Comment by iambrandonn** [comment]
Why don't you want to deploy the transpiled code?

**Comment by rojlarge** [comment]
We can certainly make do with that, it just seems unnecessarily dangerous to default to overwriting source code in-place. If it doesn't bother anyone else to do it this way then we can just add a step in our deployment process to do the transpilation, I just found it unexpected.

**Comment by iambrandonn** [comment]
Gotcha. There's an assumption that the code being transpiled has just been freshly cloned and doesn't contain any "works in progress".

**Comment by rojlarge** [comment]
Fair enough. I'm fine with leaving it as-is then, although I think it would definitely be helpful to have some additional documentation on the Production-style build process so that everyone's aware of those assumptions. Right now only running via "npm start" is covered, so just letting everyone know that they should do something else for real deployments would be a big help.

**Comment by gouldner** [comment]
I find this confusing as well.  When I check github status it tells me I have modified the code.  I can easily see a dev committing the transpiled code into github my mistake.    Is there a way to let the transpiled code get create in a different folder that can be added to gitignore ?

**Comment by gouldner** [comment]
Still no production build and deploy instructions in 1603.  Should I open a support ticket on this instead ?

**Comment by tdurkin** [comment]
@gouldner & @rojlarge - The next release will have a new npm run build option for build_babel_to which takes a parameter for target output directory. It's very similar to what Jeff has in his first comment. The release will also have more documentation around the different run options and guidance on which to use for development versus prod.

**Comment by rojlarge** [comment]
Thanks Terry! I'll keep an eye out for that in the 1604 release-- that sounds like what I was looking for.
